esmf_regrid/_esmf_sdo.py

```diff
@@ -129,6 +129,17 @@ def __init__(
         """
         self.lons = lons
         self.lats = lats
+        londims = len(self.lons.shape)
+        assert len(self.lons.shape) == londims
```

Comment:
Are these asserts necessary? you're setting the value on the line above.

Reply:
ah, good catch, I believe I may have intended this assert to be for lonbounds.